backend/src/Controllers/voterRolls.controller.ts

```diff
@@ -69,13 +84,78 @@ const editElectionRoll = async (req: any, res: any, next: any) => {
     }
 }
 
+const changeElectionRollState = async (req: any, res: any, next: any) => {
+    console.log(`-> electionRolls.changeElectionRollState ${req.election.election_id}`)
+
```

Comment:
I'd suggest a public method / API for each separate flow, based on the action being taken:
Approve Voter, Flag Voter, Reject Voter, etc,  to increase code maintainability and audit-clarity.

These can rely on a private method for updating the roll-state in the DB

Reply:
Added separate api for each flow, but kept just a single middleware for now.